base/scss/overrides/reset/_civicrm.scss

```diff
@@ -50,3 +50,62 @@ table {
 
   display: block;
 }
+
+.crm-button .crm-i {
+  left: .6em;
+  top: .6em;
+}
+
+// aligns icons inside buttons. changes from top -1 to top 2:
+.button .icon {
+  top: 2px;
+}
+
+.crm-button-type-next {
+  @include button-outline-variant(white, $brand-primary);
+
+  .crm-i { display: none; }
+}
+
+.crm-button-type-cancel,
+.crm-button-type-delete {
+  border-color: $gray-light;
+
+  input {
+    color: $text-color !important;
+  }
+}
+
+// fixes radio button's circle position
+.crm-form-radio + label {
+  &::before {
+    min-width: 16px;
+    width: 16px;
+    height: 16px;
+  }
+
+  &::after {
+    top: 8px;
+  }
+}
+
+// removes extra padding on select2 ul element added by bootstrap:
```

Comment:
Could you please try the following:
1) Temporary remove these styles that fix the selector
2) Apply a class "no-select2" to the broken selector
What happens if you do it?